woocommerce/woocommerce
# [Accessibility] Inform screen reader users when mini cart updates

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #37597 
Closes #37598 



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Open the shop page.
2. Turn on the screen reader (SR).
3. Go to a product card and click on the "Add to cart" button.
![image](https://github.com/woocommerce/woocommerce/assets/9420947/63acba7f-72a4-4a02-854b-9ae89956d642)
4. Suppose the Product's name is "Album", verify the SR announces "'Album' has been added to your cart"
5. Click on the same button again.
6. Verify the SR announces the same message again: "'Album' has been added to your cart".
7. Go to any other product and click on the "Add to cart" button".
8. Verify the SR announces the message with the name of the newly added product.
9. Go to the mini cart.
10. Remove a product.
![image](https://github.com/woocommerce/woocommerce/assets/9420947/072546e4-07da-4970-8724-aa0e26504546)
11. Assuming the removed product is the "Album" one, verify the SR announces "'Album' has been removed from your cart".
12. Remove another product and verify the message is updated with the name of the newly removed product.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
